torch/csrc/jit/passes/quantization.cpp

```diff
@@ -198,25 +198,26 @@ Node* InsertObserversHelper::insertObserverFor(
     observer_name = "_observer_" + c10::to_string(uid_++);
   }
   module.register_module(observer_name, observer);
+
   // Get handle of observer module
-  Node* observer_instance = g->create(c10::prim::GetAttr);
-  // self._observer_v
-  observer_instance->addInput(g->inputs()[0]);
-  observer_instance->s_(c10::attr::name, observer_name);
+  Node* observer_instance =
+      g->createGetAttr(g->inputs()[0], observer_name)->insertAfter(v->node());
   observer_instance->output()->setDebugName(observer_name);
-  observer_instance->output()->setType(observer.type());
-  observer_instance->insertAfter(v->node());
-
-  // Create forward method call
-  Node* call = g->create(c10::prim::CallMethod);
-  TORCH_INTERNAL_ASSERT(call != nullptr, "Failed to create forward call node");
-  call->s_(c10::attr::name, "forward");
-  call->addInput(observer_instance->output());
-  call->addInput(v);
-  call->output()->setType(v->type());
-  call->output()->setDebugName(v->debugName() + ".observed");
-  call->insertAfter(observer_instance);
-  return call;
+
+  {
```

Comment:
btw why do you put this inside a block?

Reply:
`WithInsertPoint` is an RAII guard. As a rule I add a separate block when that's used, to ensure it goes out of scope at the right time, even if someone in the future adds code after the `return` here